engine/sync_manager.go

```diff
@@ -610,6 +610,13 @@ func (m *syncManager) syncOrderbook(c *currencyPairSyncAgent, e exchange.IBotExc
 		e.SupportsREST() &&
 		time.Since(s.LastUpdated) > m.config.TimeoutWebsocket &&
 		time.Since(c.Created) > m.config.TimeoutWebsocket {
+		if w, err := e.GetWebsocket(); err != nil && w.IsConnected() {
+			// With an active websocket connection, we can assume the orderbook
+			// is being updated via the websocket connection, It could be very
+			// illiquid. There is no need to fall over to rest.
+			s.LastUpdated = time.Now()
```

Comment:
I think you can come up with a less hacky way of going about this. It hasn't been updated so I don't think you need to lie to make it not switch over.